action.yml

```
@@ -45,6 +45,13 @@ inputs:
     default: auto
   pre-test-cmd:
     description: Extra command to invoke before ansible-test
+  pull-request-change-detection:
+    description: >-
+      Whether to use change detection for pull requests. If set to `true`, will
+      use change detection to determine changed files against the target
+      branch, and will not upload code coverage results. If the invocation is
+      not from a pull request, this option is ignored.
+    default: 'false'
```

Let's keep the style aligned with other places.
```suggestion
    default: false
```

---

That didn't work, pre-commit didn't like it.

---

Ah, so maybe yamllint's config needs updating.